### DISCUSSION TITLE
Wiki Entry - Intel Celeron G440

### DISCUSSION THREAD

**Discussion by svmlegacy** [cyring/CoreFreq#341] [General]
I can't seem to find the edit button to add an entry to Wiki...

[Intel Celeron G440](https://gist.github.com/svmlegacy/0b799f2842693ad396486afb06960bb3)

**Comment by cyring** [comment]
@svmlegacy : Thanks!

No Package C-states are showing up with this SNB/Celeron

You should have received an invitation as collaborator of Wiki

(Suggest you clean up your credentials from screenshot. Images are public on GitHub)

**Reply by svmlegacy** [reply]
Good point on the screenshot. Will re-upload cleaned up image.

Possibly there is no package C-states due to it being single core? I'll add a Dual-Core Celeron G530 after it finishes a lengthy benchmark run of GCC-building.

**Reply by svmlegacy** [reply]
Follow up:

Apparently this motherboard disables package C-states when left in Auto. G440 supports a max package C-State of C3. G530 supports C6.

I've forced package max C-State to C3 to produce this:
![SR0BY Package C-States](https://user-images.githubusercontent.com/59940364/168493145-262955e9-2eb5-47a6-b773-69c753e569cb.png)

**Reply by cyring** [reply]
> Follow up:
> 
> Apparently this motherboard disables package C-states when left in Auto. G440 supports a max package C-State of C3. G530 supports C6.
> 
> I've forced package max C-State to C3 to produce this: ![SR0BY Package C-States](https://user-images.githubusercontent.com/59940364/168493145-262955e9-2eb5-47a6-b773-69c753e569cb.png)

Oh oh, well done!
